### ISSUE TITLE
[ENHANCEMENT] Inserting drop caps before selected neume would be nice

### ISSUE THREAD

**Issue by JoelSDavies** [danielgarthur/neanes#37]
**Is your feature request related to a problem? Please describe.**
At some point the drop caps feature changed from inserting the drop caps before the selected neume to inserting them after the selected neume. This is annoying because drop caps typically go at the beginning of a line, so to get them in the right spot, you need to up and over to the end of the previous line. This may even be on another page. And at the beginning of a score, you have to select the mode martyria box. Selecting the neume right where you want to put the drop cap was much easier.

**Describe the solution you'd like**
If there wasn't a particular reason for changing the spot where the drop cap is inserted, maybe just change it back? Or if there was a reason to change it, maybe make an option that users can choose which way they want it.

**Comment by danielgarthur** [comment]
When the feature was first added, I thought of drop caps as a sort of "lyric" that you add to an existing neume, which is why it was placed before the selected neume. Later when the keyboard shortcuts were added, I thought that maybe some users might think of it as being separate and would type it out as it would appear left to right: drop cap, neume, neume, etc.

Either viewpoint makes sense, so I've split the drop cap into before and after options. The button in the main toolbar uses the "before" option because I had to pick one of the two options. But the main menu has both options with shortcuts Ctrl+D and Ctrl+Shift+D.
